README.md

```diff
@@ -48,24 +48,17 @@ Bokeh
 </tr>
 </table>
 
-Bokeh is a Python interactive visualization library that targets modern
-web browsers for presentation. Its goal is to provide elegant, concise
-construction of novel graphics in the style of D3.js, but also deliver this
-capability with high-performance interactivity over very large or streaming
-datasets. Bokeh can help anyone who would like to quickly and easily create
-interactive plots, dashboards, and data applications.
+Bokeh, a Python interactive visualization library, enables beautiful and
+meaningful visual presentation of data in modern web browsers. With Bokeh,
+you can quickly and easily create interactive plots, dashboards, and data
+applications.
 
-Please visit the [Bokeh web page](http://bokeh.pydata.org/en/latest) for more information and full documentation.
-
-To get started quickly, follow the [Quickstart](http://bokeh.pydata.org/en/latest/docs/user_guide/quickstart.html) in the online documentation.
-
-Be sure to follow us on Twitter [@bokehplots](http://twitter.com/BokehPlots), as well as on
-[Vine](https://vine.co/bokehplots), and [Youtube](https://www.youtube.com/channel/UCK0rSk29mmg4UT4bIOvPYhw)!
-
-
-Interactive gallery
-===================
+Bokeh helps provide elegant, concise construction of novel graphics in the
+style of D3.js, while also delivering **high-performance** interactivity over
+very large or streaming datasets.
 
+[Interactive gallery](https://bokeh.pydata.org/en/latest/docs/gallery.html)
+===========================================================================
```

Comment:
Really huge links as section titles always look a bit off to me, maybe the second level header (i.e. `-----`) here to make the link match the other section titles below?